src/jobsautoreport/query.py

```diff
@@ -76,15 +76,21 @@ def _get_query_usages(from_date: datetime, to_date: datetime) -> dict[str, Any]:
         return {
             "query": {
                 "bool": {
-                    "must": [
+                    "filter": [
                         {
                             "range": {
                                 "usage.start_date": {
-                                    "gte": from_date,
                                     "lte": to_date,
                                 }
                             }
-                        }
+                        },
+                        {
+                            "range": {
+                                "usage.end_date": {
+                                    "gte": from_date,
```

Comment:
I'm probably missing some context, but why would the `start_date` need to be before the `to_date` and then `end_date` after the `from_date`?
Seems that (excluding bugs) every job that that started before the `to_date` would necessarily end after the `from_date`.

Could it be that dates are mixed up (start < from_date && end < to_date)?

Reply:
This is just a short way of representing intersection between intervals. any job that these two conditions hold for it must intersect with the desired interval

Reply:
So we are selecting jobs that were running in the timeframe given? Either way this is just for my curiosity :)

Reply:
Yes